admin/builders/osx64_build.sh

```
@@ -127,10 +129,8 @@ function build_component() {
 
 mkdir -p ${release_dir}
 
-build_component "percolator-noxml" "${src_dir}/percolator" "-DXML_SUPPORT=OFF"
-build_component "percolator" "${src_dir}/percolator" "-DXML_SUPPORT=ON -DGOOGLE_TEST=1"
-build_component "converters" "${src_dir}/percolator/src/converters" "-DSERIALIZE=TokyoCabinet"
+build_component "percolator" "${src_dir}/percolator" "-DGOOGLE_TEST=1"
 
 echo "build directory was : ${build_dir}";
 
-cp -v ${build_dir}/{percolator-noxml,percolator,converters}/*.pkg ${release_dir};
+cp -v ${build_dir}/percolator/*.pkg ${release_dir};
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)